pkg/kernel/networkservice/connectioncontextkernel/ipcontext/policyroute/server.go

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) 2022 Doc.ai and/or its affiliates.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build linux
+
+package policyroute
```

Comment:
Should we move this to sdk?

Reply:
I think you are right
Moved it to separate PR - https://github.com/networkservicemesh/sdk/pull/1216